nemo/collections/nlp/models/qa_model.py

```diff
@@ -128,6 +183,9 @@ def _setup_dataloader_from_config(self, cfg: DictConfig):
             mode=cfg.mode,
             use_cache=cfg.use_cache,
         )
+        if cfg.mode == "eval":
+            self.validation_dataset = dataset
+
```

Comment:
why are you adding this?

Reply:
for evaluation since dataset has the evaluation method